tests/test_hep_bd9xx.py

```diff
@@ -254,7 +254,6 @@ def test_special_collections_from_980__a():
 
     assert validate(result['special_collections'], subschema) is None
     assert expected == result['special_collections']
-    assert '_collections' not in result
```

Comment:
We could convert this test to assert things on both `special_collections` and `_collections`, but let's take the shortcut and drop the now wrong assertion on `_collections`. We already have a test below that tests the logic added by this PR.